src/shared/ptyfwd.c

```diff
@@ -89,6 +89,7 @@ struct PTYForward {
         char in_buffer[LINE_MAX], *out_buffer;
         size_t out_buffer_size;
         size_t in_buffer_full, out_buffer_full;
+        size_t out_buffer_write_len;
```

Comment:
hmm, can we find a better name for this maybe? as i understand this will point to the last non-ansi-seq char in the buffer, i.e. the maximum length that covers uncorrupted ansi sequences, right?

maybe out_buffer_uncorrupted_ansi_seq_length?

Reply:
a comment here explaining this might be good too

Reply:
Will add a comment.